e2e.yaml

```diff
@@ -0,0 +1,28 @@
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Apache Software Foundation (ASF) licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This file is used to show how to write configuration files and can be used to test.
+
+verify:
```

Comment:
Maybe this file should in `test` or `example` folder? What do u think?

Reply:
It seems we can't specify the config file now, so I put it in the root dir for testing.

Reply:
Let's put this in the `example` folder for now, if any of you have time, please consider adding `--config (-c)` to specify the config file path

Reply:
`-c` should be important, it should be added before the main repo begins to adopt this.

Reply:
> Let's put this in the `example` folder for now, if any of you have time, please consider adding `--config (-c)` to specify the config file path.

Ok, I will do it in this PR.